File: beginner/__1000__/1009/1009.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {

	string name;
	double salary, sales, bonus;

	cin >> name >> salary >> sales;
	
	bonus = (sales * 15)/ 100;

	salary += bonus;

	cout << "TOTAL = R$ " << fixed << setprecision(2) << salary << endl;

	return 0;
}

File: beginner/__1000__/1009/1009.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
	public static void main(String[] args) throws IOException {
		InputStreamReader ir = new InputStreamReader(System.in);
		BufferedReader in = new BufferedReader(ir); 

		String vendedor = in.readLine();		
		double salary = Double.parseDouble(in.readLine());
		double sales = Double.parseDouble(in.readLine());

		double bonus = (sales * 15) / 100.0;

		System.out.printf("TOTAL = R$ %.2f\n", salary + bonus);
	}
}